### ISSUE TITLE
Search not working

### ISSUE THREAD

**Issue by syaifurrizal** [vmihalachi/turbo-editor#100]
Hi, I just purchased this app at Google Play Store. It's awesome. But I found that when I use "search", it's not working. Please fix it asap. I think this is should be a main feature that must be work. 

Regards,

**Comment by twindad** [comment]
Hi Vlad, yes, this appears to true.  I have pro version 2.2, updated today.

Neither "Find" nor find/replace works.  The app can't find the old word, so it can't replace it with the new word either.

Thanks, and kind regards ~~

**Comment by d10** [comment]
Yep, this happens. But sometimes when I try a second time, it works.

**Comment by wrbird** [comment]
Turbo Editor splits large files into "pages" and only loads one page at a time in memory and therefore can only search that one page. I wonder if this is the issue you're having?

**Comment by syaifurrizal** [comment]
Yes maybe. Are you the developer? I don't know about programing, but how if
at the search mode, the app automatically disable the page splits (I guest
this for rendering the code colors?) and leave the document as a plain text
so it's search able for large documents without hurting the memories.

Min, 21 Jun 2015 05.57 wrbird <notifications@github.com> menulis:

**Comment by wrbird** [comment]
I'm not the developer so I can't address your suggestion of how the search should work.  In my opinion the search feature is worthless.  After the last Turbo Editor update the other day I'm having so many issues with the app I had to abandon it.  And I had just purchased it, that's a bummer.  I'm trying out Quota right now on vary large HTML files and so far I have been very happy.  Search feature works as it should, syntax highlighting works, and it's fast.  I don't know what Quota is doing to allow it to be so fast compared to the other text/code editors I've tried, but it's great.

**Comment by syaifurrizal** [comment]
Good advice. I'll tries Quoda.

Btw I still have a hope with this app. I just love the UX. :D

Min, 21 Jun 2015 10.43 wrbird <notifications@github.com> menulis:

**Comment by vmihalachi** [comment]
@syaifurrizal do you have any screenshots? Mine works :)
@wrbird let's solve this issue and not write about other apps, ok?

**Comment by syaifurrizal** [comment]
Hi Vlad, Finally is you,

Here the screenshots. I write the texts at search input but it's not bring
me anywhere.

Regard,

Sel, 23 Jun 2015 16.01 Vlad Mihalachi <notifications@github.com> menulis:

> @syaifurrizal <https://github.com/syaifurrizal> do you have any
> screenshots? Mine works :)
> @wrbird <https://github.com/wrbird> let's solve this issue and not write
> about other apps, ok?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/vmihalachi/turbo-editor/issues/100#issuecomment-114412050>
> .
>
-- 

Regard,
Syaifur Rizal

**Comment by vmihalachi** [comment]
@syaifurrizal I do not see them.
